.github/workflows/master.yml

```diff
@@ -147,8 +147,8 @@ jobs:
         sudo apt-get install -y libcurl4-openssl-dev pandoc
     - name: Install packages
       run: |
-        pip install sphinx mkdocs numpy
-        gem install jekyll jekyll-redirect-from rouge
+        pip install -r ./dev/requirements-pinned.txt
```

Comment:
@HyukjinKwon - I can't find your comment anymore. Would you still like this renamed to `requirements-frozen.txt`?